src/util/misc.ts

```diff
@@ -173,25 +173,25 @@ export function saveStaticFiles(config: Config, zimCreator: ZimCreator) {
   ]);
 }
 
-export function cssPath({ output: { dirs } }: Config, css: string) {
-  return [dirs.style, `${dirs.styleModules}/${css.replace(/(\.css)?$/, '')}.css`].join('/');
+export function cssPath(css: string, subDirectory = '') {
+  return `${subDirectory ? `${subDirectory}/` : ''}${css.replace(/(\.css)?$/, '')}.css`;
 }
-export function jsPath({ output: { dirs } }: Config, js: string) {
+export function jsPath(js: string, subDirectory = '') {
   const path = (isNodeModule(js)) ? normalizeModule(js) : js;
-  return [dirs.javascript, `${dirs.jsModules}/${path.replace(/(\.js)?$/, '')}.js`].join('/');
+  return `${subDirectory ? `${config.output.dirs.mediawiki}/` : ''}${path.replace(/(\.js)?$/, '')}.js`;
```

Comment:
Do we still need to check `subDirectory`?

Reply:
same as above for `/`